games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+inherit desktop
+
+DESCRIPTION="A Mame and Retrogaming Rom Manager, Java alternative to ClrMamePro"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip -> ${P}.zip"
+QA_PREBUILT="*"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=virtual/jre-1.8"
+DEPEND="${DEPEND}"
+
+src_prepare() {
+	eapply_user
```

Comment:
Couldn't this be replaced with 'default'?

Reply:
repoman complained about missing eapply_user, but if you tell me that it can be replaced by 'default'...

Reply:
Just having src_prepare() with the sed below will mean that no user patches will be applied. But if you call default first, then it will apply user patches (call epaply_user) and then execute your sed. Try it for yourself :)